rcore/flash.c

```diff
@@ -286,21 +289,20 @@ uint32_t flash_get_resource_address(uint16_t slot_id)
 
 const File fs_find_file(const char *name)
```

Comment:
maybe have this populate into a pointer passed in?  signature something like:

`int fs_open(struct file *file, const char *name);`

Reply:
yup, that would make result explicit